src/run.rs

```diff
@@ -119,11 +128,9 @@ fn run_impl() -> anyhow::Result<i32> {
         return Ok(1);
     }
 
-    if fake_ninja_compat {
-        if matches.opt_present("version") {
-            println!("1.10.2");
-            return Ok(0);
-        }
+    if fake_ninja_compat && matches.opt_present("version") {
```

Comment:
This seems unrelated

Reply:
> This seems unrelated

yes, it's a clippy fix/suggestion. Would you like it removed from this PR?

Reply:
i've removed it